### PR TITLE
Lighting Group organization

### PR DESCRIPTION
**Pull request by biotvandenberg**
Added new LGRP type and moved the LGRP types that were in LT.yaml in to LGRP.yaml

### CODE REVIEW

**Comment by biotvandenberg** [comment]
@googlebot I signed it!

File: ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml [hunk 1]
```diff
@@ -20,4 +20,79 @@ LGRP_DC:
   is_canonical: true
   implements:
     - LGRP
-    - DC
\ No newline at end of file
+    - DC
+
+LGRP_OS_SCZ:
```
Comment (tasodorff) [inline ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:25]: @biotvandenberg did you change anything here? it looks identical to what existed before.

**Review by tasodorff — COMMENTED** [review]
Confirm that the changes here are substantive or remove them. If they are, please indicate how.

File: ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml (hunk 1 above)
Comment (biotvandenberg) [reply ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:25]: LGRP_OS_SCZ didn't exist before

File: ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml [hunk 2]
```diff
@@ -20,4 +20,79 @@ LGRP_DC:
...
+  id: ""
+  description: "Lighing group control with occupancy and zone scene control"
+  is_canonical: true
+  implements:
+  - LGRP
+  - OS
+  - SCZ
+
+LGRP_OS_LC_VADC:
+  id: "4599945020949135360"
```
Comment (tasodorff) [inline ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:35]: Did this one exist before? its showing up as new; if it is new, please remove the ID. No new type should have an ID. The system sets this.

Comment (tasodorff) [reply ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:35]: This applies to lines 34 to 98

**Review by tasodorff — COMMENTED** [review]
Clarify please

File: ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml (hunk 2 above)
Comment (biotvandenberg) [reply ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:35]: These canonical types (all are LGRP abstract types) were all erroneously place in the LT.yaml file.  I removed them from LT.yaml and moved in to LGRP.yaml.   Assume this means that the ids are all still valid.

Comment (tasodorff) [reply ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:35]: if they were simply moved, then yes. move them back and leave the IDs as they were.

Comment (biotvandenberg) [reply ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:35]: Move them back to LT.yaml?  Isn't that confusing?  I was looking for LGRP canonical types and stumbled in to them in the LT file.

Comment (tasodorff) [reply ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:35]: oh sorry i was confused. yes, leave them in the new file with the existing type ids.

**Review by tasodorff — APPROVED** [review]

**Pull request merged**